components/style/values/specified/font.rs

```diff
@@ -977,33 +977,25 @@ impl Parse for MozScriptMinSize {
 }
 
 #[cfg_attr(feature = "gecko", derive(MallocSizeOf))]
-#[derive(Clone, Copy, Debug, PartialEq)]
+#[derive(Clone, Copy, Debug, PartialEq, ToCss)]
 /// Changes the scriptlevel in effect for the children.
 /// Ref: https://wiki.mozilla.org/MathML:mstyle
 ///
 /// The main effect of scriptlevel is to control the font size.
 /// https://www.w3.org/TR/MathML3/chapter3.html#presm.scriptlevel
 pub enum MozScriptLevel {
-    /// Change `font-size` relatively
+    /// Change `font-size` relatively.
     Relative(i32),
-    /// Change `font-size` absolutely
+    /// Change `font-size` absolutely.
+    ///
+    /// Should only be serialized by presentation attributes, so even though
+    /// serialization for this would look the same as for the `Relative`
+    /// variant, it is unexposed, so no big deal.
```

Comment:
Maybe add `#[css(function)]` and name it `ServoAbsolute` or something, so that it serialises to `-servo-absolute`? At least if it ever leak outside we will know what's happening.

Reply:
Done (with `-moz-absolute` since this is moz-only anyway).